src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/BlobUtilities.cs

```diff
@@ -271,5 +287,60 @@ internal static void ValidateRange(int bufferLength, int start, int byteCount)
                 Throw.ArgumentOutOfRange(nameof(byteCount));
             }
         }
+
+        internal static int GetUserStringByteLength(int characterCount)
+        {
+            return characterCount * 2 + 1;
+        }
+
+        internal static byte GetUserStringTrailingByte(string str)
+        {
+            // Write out a trailing byte indicating if the string is really quite simple
```

Comment:
Will fix. Just kept what was there before.


---
In reply to: [63402007](https://github.com/dotnet/corefx/pull/8419#discussion_r63402007) [](ancestors = 63402007)